public/css/main.css

```diff
@@ -15,7 +15,7 @@
 }
 
 /*RESETS*/
-:root
+:root, html, body
```

Comment:
Was macht denn `:root` ? :-)

Reply:
:root is a pseudo-class selector. We define CSS Variables (= Custom Properties) on it, so that they are accessible to all elements in the document/across documents.